src/python/gudhi/tensorflow/_layer.py

```diff
@@ -0,0 +1,16 @@
+import tensorflow
+# The sklearn parse_version function is taken from 'packaging' and this dependency is not mandatory
+from sklearn.utils.fixes import parse_version
+
+class TensorflowKerasLayer(tensorflow.keras.layers.Layer):
+    """
+    TensorFlow Keras layer
+    """
+    def __init__(self, **kwargs):
+        """
+        Constructor for the TensorflowKerasLayer class
+        """
+        # On tensorflow < 2.15.1 set dynamic argument to True
```

Comment:
Did they really change that in a patch release, i.e. 2.15.0 wants `dynamic` but 2.15.1 rejects it? That's surprising :disappointed: